vividus-plugin-excel/src/main/java/org/vividus/excel/ExcelSheetWriter.java

```diff
@@ -21,17 +21,28 @@
 import java.io.OutputStream;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 import java.util.stream.IntStream;
 
+import org.apache.commons.lang3.EnumUtils;
+import org.apache.poi.ss.usermodel.Cell;
 import org.apache.poi.ss.usermodel.Row;
 import org.apache.poi.xssf.usermodel.XSSFSheet;
 import org.apache.poi.xssf.usermodel.XSSFWorkbook;
 import org.jbehave.core.model.ExamplesTable;
+import org.vividus.model.CellType;
+import org.vividus.util.DateUtils;
 
 public final class ExcelSheetWriter
```

Comment:
let's convert it to Spring bean (we can avoid passing `dateUtils` to each method, but rather inject this object in the bean)